src/highlighters.cc

```diff
@@ -1013,11 +1013,14 @@ std::unique_ptr<Highlighter> show_whitespaces_factory(HighlighterParameters para
           { "tabpad", { true, "" } },
           { "spc", { true, "" } },
           { "lf", { true, "" } },
-          { "nbsp", { true, "" } } },
+          { "nbsp", { true, "" } },
+          { "only-trailing", { false, "" } } },
         ParameterDesc::Flags::None, 0, 0
     };
     ParametersParser parser(params, param_desc);
 
+    // ToDo: I think there must be a better way to transform it into a boolean
+    bool only_trailing = parser.get_switch("only-trailing") ? true : false;
```

Comment:
try this: `bool only_trailing = (bool)parser.get_switch("only-trailing")`

Reply:
and I'd get rid of the variable and just move the `(bool)parser.get_switch("only-trailing")` directly to the function call on line 1037

Reply:
:ok_hand: